automated-comments/go/raindrops/remove_extra_bool.md

```diff
@@ -0,0 +1,2 @@
+An extra variable `%{name}` is used here to see if any of the cases applied.
+Instead a check if the string is still empty could be used.
```

Comment:
"Instead you could check..." sounds a little more engaging.

Reply:
We try to avoid "you".